src/Tribe/Views/V2/Customizer/Section/Global_Elements.php

```diff
@@ -0,0 +1,510 @@
+<?php
+/**
+ * The Events Calendar Customizer Section Class
+ * Global Elements
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Customizer\Section;
+/**
+ * Global Elements
+ *
+ * @since TBD
+ */
+final class Global_Elements extends \Tribe__Customizer__Section {
+
+	/**
+	 * ID of the section.
+	 *
+	 * @since TBD
+	 *
+	 * @access public
+	 * @var string
+	 */
+	public $ID = 'global_elements';
+
+	/**
+	 * Allows sections to be loaded in order for overrides.
+	 *
+	 * @var integer
+	 */
+	public $queue_priority = 15;
+
+	/**
+	 * Placeholder for filtered multiplier for small font size.
+	 *
+	 * @var float
+	 */
+	private $small_font_multiplier = .75;
+
+	/**
+	 * Placeholder for filtered multiplier for large font size.
+	 *
+	 * @var float
+	 */
+	private $large_font_multiplier = 1.5;
+
+	/**
+	 * Placeholder for filtered min font size.
+	 *
+	 * @var float
+	 */
+	private $min_font_size = 8;
+
+	/**
+	 * Placeholder for filtered max font size.
+	 *
+	 * @var float
+	 */
+	private $max_font_size = 172;
+
+	/**
+	 * This method will be executed when the Class is Initialized.
+	 *
+	 * @since TBD
+	 */
+	public function setup() {
+		parent::setup();
+
+
+		/**
+		 * Allows users and plugins to change the "small" font size multiplier.
+		 *
+		 * @since TBD
+		 *
+		 * @param int $small_font_multiplier The multiplier for "small" font size.
+		 *
+		 * @return int The multiplier for "small" font size. This should be less than 1.
+		 */
+		$this->small_font_multiplier = apply_filters( 'tribe_customizer_small_font_size_multiplier', $this->small_font_multiplier );
+
+		/**
+		 * Allows users and plugins to change the "large" font size multiplier.
+		 *
+		 * @since TBD
+		 *
+		 * @param int $large_font_multiplier The multiplier for "large" font size.
+		 *
+		 * @return int The multiplier for "large" font size. This should be greater than 1.
+		 */
+		$this->large_font_multiplier = apply_filters( 'tribe_customizer_large_font_size_multiplier', $this->large_font_multiplier );
+
+		/**
+		 * Allows users and plugins to change the minimum font size.
+		 *
+		 * @since TBD
+		 *
+		 * @param int $min_font_size The enforced minimum font size.
+		 *
+		 * @return int The enforced minimum font size.
+		 */
+		$this->min_font_size = apply_filters( 'tribe_customizer_minimum_font_size', 8 );
+
+		/**
+		 * Allows users and plugins to change the maximum font size.
+		 *
+		 * @since TBD
+		 *
+		 * @param int $min_font_size The enforced maximum font size.
+		 *
+		 * @return int The enforced maximum font size.
+		 */
+		$this->max_font_size = apply_filters( 'tribe_customizer_maximum_font_size', 72 );
```

Comment:
pass in the variable?

```suggestion
		$this->max_font_size = apply_filters( 'tribe_customizer_maximum_font_size', $this->max_font_size );
```